draft-ietf-webtrans-http3.md

```diff
@@ -147,6 +147,12 @@ MUST NOT process any incoming WebTransport requests until the client settings
 have been received, as the client may be using a version of WebTransport
 extension that is different from the one used by the server.
 
+In addition to the setting above, the server MUST send a
+SETTINGS_MAX_WEBTRANSPORT_SESSIONS parameter indicating the maximum number of
```

Comment:
I thought we were going to only have one setting since SETTINGS_MAX_WEBTRANSPORT_SESSIONS=0 means the same as "WebTransport not supported".

Reply:
We can't do that at this point, since only the server sends SETTINGS_MAX_WEBTRANSPORT_SESSIONS, and we need a setting from the client to do version negotiation.

Reply:
Can't we use only the header for VN and drop the setting? If anyone is implementing draft versions with an intermediary they can parse the headers.

Reply:
> > > Additionally, this PR addresses #34 but not #22 so it shouldn't be marked as fixing #22.
> > 
> > 
> > I thought we've agreed at some point that this is all we want in terms of solving #22?
> 
> I don't recall that (but I could be misremembering), do you have a reference to point me to?

I checked the meeting notes for the design team, and it looks like we decided the opposite of what I remember, so it looks like I'm the one misremembering here.

Reply:
From what I remember from the way our code works, that would work poorly, so I'm hesitant to do that.

Reply:
This is written such that MAX_WEBTRANSPORT_SESSIONS is only sent by the server, do we want to have enable_webtransport (or version) be still sent by both/the client, or do we think that using the upgrade token indicates support on the part of the client? (And is that easy enough to handle in code)